### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by luisrpavanello**
- Document work in a professional way.
- Create a new GitHub repo.
- Create a new branch according to the GitHub flow rules.
- Add a .gitignore file.
- Add node_modules to gitignore file.
- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project - please use this template but remember to customize it to your project.
  
  Add 3 files:
- html file that includes a header with the text "Hello Microverse Project!". Use h1 tag for that.
- css file that includes one class with styles for your header (make it in your favorite color).
- test.md file with the text "This file should be ignored by git".
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
- Open a pull request.

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @luisrpavanelli, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ]  I would implore you to add always add a descriptive Pull Request summary describing the changes that you implemented in your pull request so that the reviewer will understand the things you did in the PR. A descriptive PR includes a heading and all the things you've done listed under it.

Writing appreciation as a pull request summary is not professional. So highlight the features you implemented instead 👍🏽 

- [ ] The link to your MIT license in your readme is broken. KIndly add the link to the MIT link or you create a `MIT.md` file including your license text and link it to your readme so as to make your repo more professional 👍🏽 

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — COMMENTED** [review]
# Hello @luisrpavanelli   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
## STATUS: APPROVED
## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by luisrpavanello — COMMENTED** [review]
Modify the line link, I have left a few comments that should be addressed before this gets merged

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@
 
 ## 🚀 View <a name="view"></a>
 
-- [Click here to view]https://htmlpreview.github.io/?https://github.com/luisrpavanelli/Hello-Microverse-Project/blob/main/index.html
+[Click here to view]: https://htmlpreview.github.io/?https://github.com/luisrpavanelli/Hello-Microverse-Project/blob/main/index.html
```
Comment (luisrpavanello) [inline README.md:69]: Modify the line link, I have left a few comments that should be addressed before this gets merged

**Review by luisrpavanello — COMMENTED** [review]
Modify the line link, I have left a few comments that should be addressed before this gets merged

**Review by joskalenda — COMMENTED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @luisrpavanelli , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**